lms/djangoapps/instructor/views/masters_gradebook_api.py

```diff
@@ -0,0 +1,128 @@
+"""
+Grade book view for instructor and pagination work (for grade book)
+which is currently use by ccx and instructor apps.
+"""
+import math
+
+from django.contrib.auth.models import User
+from django.db import transaction
+from django.views.decorators.cache import cache_control
+from opaque_keys.edx.keys import CourseKey
+
+from edxmako.shortcuts import render_to_response
+from lms.djangoapps.courseware.courses import get_course_with_access
+from lms.djangoapps.grades.course_grade_factory import CourseGradeFactory
+from lms.djangoapps.instructor.views.api import require_level
+from xmodule.modulestore.django import modulestore
+
+# Grade book: max students per page
+MAX_STUDENTS_PER_PAGE_GRADE_BOOK = 20
+
+
+def calculate_page_info(offset, total_students):
```

Comment:
I don't think we'll use this function outside of `get_grade_book_page`, which we also don't need, so we could delete it.